### DISCUSSION TITLE
Plans for Python 3.9 support?

### DISCUSSION THREAD

**Discussion by slieb** [ironmussa/Optimus#1193] [Q&A] [answered]
I tried doing an install of optimus on my MacBook Pro (macOS V 11.5.1) under Python 3.9.1 but  got lots of error messages.  Requirements say Python 3.7 or 3.8, so just wondering if there are plans or an ETA for Python 3.9 support?

Thanks.

**Comment by argenisleon** [comment, marked as answer]
Hi @slieb,
No, we have no plans to support python 3.9 yet, but, I am going to start exploring this.
